test/cli_run_with_privileged_test.go

```diff
@@ -0,0 +1,76 @@
+package main
+
+import (
+	"github.com/alibaba/pouch/test/command"
+	"github.com/alibaba/pouch/test/environment"
+
+	"github.com/go-check/check"
+	"github.com/gotestyourself/gotestyourself/icmd"
+)
+
+// PouchRunPrivilegedSuite is the test suite for run CLI.
+type PouchRunPrivilegedSuite struct{}
+
+func init() {
+	check.Suite(&PouchRunPrivilegedSuite{})
+}
+
+// SetUpSuite does common setup in the beginning of each test suite.
+func (suite *PouchRunPrivilegedSuite) SetUpSuite(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+
+	environment.PruneAllContainers(apiClient)
+
+	PullImage(c, busyboxImage)
+}
+
+// TearDownTest does cleanup work in the end of each test.
+func (suite *PouchRunPrivilegedSuite) TearDownTest(c *check.C) {
+}
+
+// TestRunWithPrivilege is to verify run container with privilege.
+func (suite *PouchRunPrivilegedSuite) TestRunWithPrivilege(c *check.C) {
+	name := "run-privilege"
+
+	res := command.PouchRun("run", "--name", name, "--privileged",
```

Comment:
Just double check whether command `brctl` is the correct one to cover `privileged ` functionality.

To see the value of comparison, I suggest that we could add a test without `--privileged` and execute the same command `brctl` to test whether it is indeed returning an error. Still in `TestRunWithPrivilege `

WDYT? @HusterWan